client/signup/steps/site-advance-info/index.jsx

```diff
@@ -0,0 +1,31 @@
+import { useTranslate } from 'i18n-calypso';
+import StepWrapper from 'calypso/signup/step-wrapper';
+
+function SiteAdvanceInfo() {
+	return <p>Please provide Site Advance Info</p>;
+}
+
+export default function WrapperSiteInformationCollection( props ) {
+	const { flowName, stepName, positionInFlow } = props;
+	const translate = useTranslate();
+
+	const headerText = translate( 'Site Advance Info' );
+	const subHeaderText = translate( 'We need some Advance details to build your site.' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'We need some basic details to build your site.' )` *ES Score: 6*